### DISCUSSION TITLE
Wall Mounted Cube immersed into the Atmospheric Boundary Layer

### DISCUSSION THREAD

**Discussion by leocotteleer** [xcompact3d/Incompact3d#238] [Q&A] [answered]
Hi all, 

I am trying to run a LES for a Wall mounted cube immersed into the ABL with a precursor simulation providing the appropriate turbulent inflow. For the cube's geometry, I used the xcompact-tooblox and the genepsi tool but the ABL parameters are not available in the native python package. 

Do you have any suggestion on how to proceed ? If i used the toolbox, can I nevertheless impose the conditions into a separated input.i3d file and the solver will understand that there are an input file and a geometry folder ?

Or should I create the geometry with another tool?

Thanks in advance !

**Comment by slaizet** [comment, marked as answer]
Hi,

The toolbox is not yet compatible with ABL capabilities. My suggestion would be to hard code the geometry (similar to how it is done for a cylinder.

I shall also say that it is likely that you would need a wall model at the walls of your immersed cube. This is something we are working on at the moment, and will hopefully be a new functionality available soonish (it is probably going to take few months).

Best,

Sylvain
